src/status_im/contexts/communities/actions/community_options/view.cljs

```diff
@@ -112,7 +112,7 @@
   {:icon                :i/share
    :accessibility-label :share-community
    :label               (i18n/label :t/share-community)
-   :on-press            #(hide-sheet-and-dispatch [:communities/share-community-pressed id])})
+   :on-press            #(rf/dispatch [:communities/share-community-pressed id])})
```

Comment:
Changed this to only dispatch because we're opening a share-sheet with this action and calling `hide-sheet-and-dispatch` will also hide the share-sheet.